.github/.env

```diff
@@ -0,0 +1 @@
+CURRENT_OSM_VERSION=v0.9.0
```

Comment:
Can we kill this? Each individual component should have a version number either in the component's json or in the folder/filename (ideally in the json as this needs to be included in the capabilities registry in meshery server). @Shreyas220 @DelusionalOptimist @sayantan1413 @utkarsh-pro @Revolyssup

Reply:
Yes @leecalcote we can kill this. 
This was the hastiest idea I could come up to make a workflow trigger consistently with a service mesh's new releases.
Just to clarify, we are storing service mesh version in component's schema even right now and probably the appropriate approach will be reading them from the schema itself for triggering component generation.